forms/HtmlEditorField.php

```diff
@@ -674,7 +680,9 @@ protected function getFieldsForOembed($url, $file) {
 					'center' => _t('HtmlEditorField.CSSCLASSCENTER', 'Centered, on its own.'),
 					'left' => _t('HtmlEditorField.CSSCLASSLEFT', 'On the left, with text wrapping around.'),
 					'right' => _t('HtmlEditorField.CSSCLASSRIGHT', 'On the right, with text wrapping around.')
-				)
+				),
+				Config::inst()->get('HtmlEditorField', 'media_alignment')
```

Comment:
> BTW is there any way to view all the available config settings in SilverStripe? I feel like they should be exposed automatically in the API docs but the only way I know to find these kind of things is to go digging in the source code.


Not yet but I'm hoping to get something built for api.silverstripe.org that will let us extract them.